src/Raven.Server/ServerWide/Maintenance/ClusterObserver.cs

```diff
@@ -137,28 +137,28 @@ public void Run(CancellationToken token)
 
         private readonly Dictionary<string, long> _lastLogs = new Dictionary<string, long>();
 
-        private void LogMessage(string message, Exception e = null, bool info = true)
+        private void LogMessage(string message, Exception e = null, bool info = true, string database = null)
         {
             if (_iteration % 10_000 == 0)
                 _lastLogs.Clear();
 
-            if (_logger.IsInfoEnabled || _logger.IsOperationsEnabled)
+            if (_lastLogs.TryGetValue(message, out var last))
             {
-                if (_lastLogs.TryGetValue(message, out var last))
-                {
-                    if (last + 10 > _iteration)
-                        return;
-                }
-                _lastLogs[message] = _iteration;
+                if (last + 60 > _iteration) // 30 sec
```

Comment:
I don't follow this computation?

Reply:
each iteration happened every 500 ms.
500ms * 60 = 30 s

Reply:
Can you add this as a comment?

Reply:
done